<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="PINKYPRIYA" ignored="3" name="Default test(failed)" tests="2" failures="2" timestamp="04 Nov 2020 12:52:50 GMT" time="0.132" errors="0">
  <testcase name="test3" time="0.026" classname="org.rerun.Employee">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.rerun.Employee.test3(Employee.java:19)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase name="test2" time="0.002" classname="org.rerun.Employee">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.rerun.Employee.test2(Employee.java:14)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase name="test1" time="0.0" classname="org.rerun.Employee">
    <ignored/>
  </testcase> <!-- test1 -->
  <testcase name="test5" time="0.0" classname="org.rerun.Employee">
    <ignored/>
  </testcase> <!-- test5 -->
  <testcase name="test4" time="0.0" classname="org.rerun.Employee">
    <ignored/>
  </testcase> <!-- test4 -->
</testsuite> <!-- Default test(failed) -->
